framework/test/jest/unit/specs/modules/chain/bft/bft_finality_manager_protocol_specs.spec.js

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const scenario4DelegatesMissedSlots = require('./bft_specs/4_delegates_missed_slots.json');
```

Comment:
@nazarhussain are this require paths wrong? They're not pointing to `protocol-specs` folder, where the specs actually are.

Reply:
@limiaspasdaniel The folder which was generated by protocol-spec is symlinked to tests folders. So if we want to extract the specs out of this repo we can simply copy over the files with out code change.